File: 程式語言 V2答案檔/PHP/307.php
<?php 
    function compute($array){
        $max_value = 0;
        for($i = 0; $i < 5; $i++){
            $max_value = max($max_value, $array[$i]);
        }
        return $max_value;
    }
    $array = new SplFixedArray(5);
    for($i = 0; $i < 5; $i++){
        $array[$i] = intval(fgets(STDIN));
    }
    $max_value = compute($array);
    printf("%d\n", $max_value);
?>

File: Programming Language(ver.2)/Go/307.go
package main
import "fmt"

func compute(array [5]int) int {
	max := 0
	for i := 0; i < 5; i++{
		if array[i] >= max{
			max = array[i]
		}
	}
	return max
}

func main(){
	var array [5]int
	for i := 0; i < 5; i++{
		fmt.Scanln(&array[i])
	}
	fmt.Println(compute(array))
}
